src/ripple/app/misc/TxQ.h

```diff
@@ -306,7 +306,7 @@ class TxQ
 
     /** Return the next sequence that would go in the TxQ for an account. */
     SeqProxy
-    nextQueuableSeq(std::shared_ptr<SLE const> const& sleAccount) const;
+    nextQueuableSeq(std::optional<AcctRootRd const> const& acctRoot) const;
```

Comment:
Can we replace the `std::optional` with a `AcctRootRd const&`? I can't imagine it ever makes sense to ask for a queuable seq for a non-existant account root. I understand the old interface took a pointer type - and it probably should not have.

There are other functions that are taking a `optional<AcctRoot...>`, can we audit these as well to see if we can remove the `optional`?